manifests/templates/reconciler-manager-configmap.yaml

```diff
@@ -67,6 +67,9 @@ data:
            volumeMounts:
            - name: repo
              mountPath: /repo
+           - name: service-account
```

Comment:
Suggest adding a comment to explain why we need to override the automounted service account info.

Reply:
Done